packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -58,13 +58,13 @@ class VmServiceWrapper implements VmService {
     }
   }
 
-  Uri get connectedUri => _connectedUri;
-  final Uri _connectedUri;
+  Uri? get connectedUri => _connectedUri;
+  final Uri? _connectedUri;
 
   final bool trackFutures;
   final Map<String, Future<Success>> _activeStreams = {};
 
-  final Set<TrackedFuture<Object>> activeFutures = {};
+  final Set<TrackedFuture<Object?>> activeFutures = {};
```

Comment:
why does object need to be nullable

Reply:
this is final to be nullable. The type returned by the future doesn't matter.